services/queue/src/queueservice.js

```diff
@@ -388,8 +385,7 @@ class QueueService {
     // Construct queue name prefix (add priority later)
     let namePrefix = [
       this.prefix, // prefix all queues
-      hashId(provisionerId), // hash of provisionerId
-      hashId(workerType), // hash of workerType
```

Comment:
This will change the name of the pending queues, meaning that tasks pending when the upgrade is made will never start.  I think this will have to split the taskQueueId and hash individually.  Moving that to a utility function would make it easy to keep compatibility when we eventually drop the requirement that a taskQueueId contain `/`.

Reply:
Note taken! I was not aware of the implications of this.